src/_help/bump-cli.md

```diff
@@ -28,164 +35,205 @@ Or, with Yarn via
 
-## Using Bump CLI
+## Usage
 
-To list all the available commands, just type `bump` in your command line environment. You can get some help anytime by using `--help` on any command. Example: `bump deploy --help` .
+To list all the available commands, just type `bump` in your command line environment.
 
-## Preview a file
+```shell
+$ bump --help
+The Bump.sh CLI is used to interact with your API documentation hosted on Bump.sh by using the API of developers.bump.sh
 
-When writing a documentation, you'll want to preview how it renders on Bump.sh. This is exactly the goal of the `preview` command: it will create a temporary documentation with a unique URL, which will be available for a short period of time.
+VERSION
+  bump-cli/2.7.2 linux-x64 node-v16.17.0
 
-Usage from a local OpenAPI or AsyncAPI file
+USAGE
+  $ bump [COMMAND]
 
-```shell
-bump preview path/to/file.json
+COMMANDS
+  deploy   Create a new version of your documentation from the given file or URL.
+  diff     Get a comparison diff with your documentation from the given file or URL.
+  help     Display help for bump.
+  preview  Create a documentation preview from the given file or URL.
 ```
 
-You can also preview a file available from a URL
+ You can also get some help anytime by adding `--help` to any command. Example: `bump deploy --help`.
+
+## Prepare your Bump.sh account
+
+While some commands don't need any API token (`preview` or `diff`) you will need an access key if you want to interact with your Bump.sh documentation.
+
+Head to your Documentation settings in the “CI deployment” section, or your Account or Organization settings in the “API keys” section to fetch a personal token for later usage.
+
+## Commands
+
+* [`bump deploy [FILE]`](#bump-deploy-file)
+* [`bump diff [FILE]`](#bump-diff-file)
+* [`bump preview [FILE]`](#bump-preview-file)
+
+### `bump deploy [FILE]`
+
+When you update your API, you want its documentation to be live for your API users. This is what the deploy command is for.
 
 ```shell
-bump preview https://developers.bump.sh/source.yaml
+bump deploy path/to/api-document.yml --doc my-documentation --token $DOC_TOKEN
 ```
 
-### Live preview
+> You can find your own `my-documentation` slug and `$DOC_TOKEN` api key from your [documentation settings](https://bump.sh/docs).
+{: .info}
 
-By using the `--live` flag you can stay focused on API design (OpenAPI or AsyncAPI file) while seeing a continuously updated preview each time you save your API definition file.
-
-- Launch the live preview command in your terminal
+You can also deploy a given API document to a different branch of your documentation with the `--branch <branch-name>` parameter. Please note the branch will be created if it doesn’t exist. More details about the branching feature is available on [this dedicated help page](https://docs.bump.sh/help/branching). E.g. deploy the API document to the `staging` branch of the documentation:
```

Comment:
```suggestion
You can also deploy a given API document to a different branch of your documentation with the `--branch <branch-name>` parameter. Please note that the branch will be created if it doesn’t exist. More details about the branching feature are available on [this dedicated help page](https://docs.bump.sh/help/branching). E.g., deploy the API document to the `staging` branch of the documentation:
```

Reply:
ok for the `are` not sure about the other two suggestions